components/ArticleMenu/ArticleMenu.tsx

```diff
@@ -164,7 +176,10 @@ const ArticleMenu = ({
               leaveFrom="transform opacity-100 scale-100"
               leaveTo="transform opacity-0 scale-95"
             >
-              <Popover.Panel className="origin-top-right absolute bottom-14 right-0 lg:left-16 lg:bottom-0 mt-2 w-48 rounded-md shadow-lg py-1 bg-white dark:bg-white ring-1 px-1 ring-black ring-opacity-5 focus:outline-none">
+              <Popover.Panel
```

Comment:
Rather than styling, do you think this might be a cleaner solution? https://headlessui.com/react/popover#closing-popovers-manually This lets you use the internal popover state to close things. 

Let me know what you think!